chrome/content/scaffold/scaffold.js

```diff
@@ -143,14 +158,15 @@ var Scaffold = new function () {
 				_editors.importGlobal = monaco;
 				_editors.import = editor;
 			}),
-			codeWin.loadMonaco({ language: 'javascript' }).then(({ monaco, editor }) => {
+			codeWin.loadMonaco({ language: 'javascript', emptyOverlayMessage: Zotero.getString('scaffold-empty-editor-code') }).then(({ monaco, editor }) => {
```

Comment:
We don't really need to add `scaffold.ftl` for this - we generally just hardcode English strings for Scaffold. US English is the only supported locale for translator dev. (Which is probably something we should mention in the docs.)